### DISCUSSION TITLE
현재 작업 프로세스에 관해서

### DISCUSSION THREAD

**Discussion by fkdl0048** [The-Day-In-1984/1984_Project#18] [Show and tell]
- #17 

일단 생각나는 대로 정리를 해봤는데 너무 혼자 막 정한 게 아닌지 걱정이 되네요..  

일단 틀을 잡는 게 중요하다고 생각해서 내용을 적어봤는데 조금 무리거나 변경되었으면 하는 내용은 가감 없이 말해주세요.

**Comment by ccmuk58** [comment]
좋은 가이드 감사합니다.
진행하면서 개선 방안 있으면 제시하겠습니다.
